### ISSUE TITLE
Feature request: full weather report for next few days

### ISSUE THREAD

**Issue by BrendanTCC** [tjmarkham/win10widgets#98]
Hey,
It would be pretty cool if you added the ability to click on a particular days weather, and the weather widget pulls the weather info for that day up! =)

**Comment by wtspace** [comment]
I agree! I was just looking at Weather Underground's layout. [Weather Underground 10-Day](https://www.wunderground.com/forecast/us/ny/albany?cm_ven=localwx_10day).

I think it would look great with the Windows 10 style. I also would really like the customized features on Wunderground's site. Features like cloud coverage, prec. accumulation, etc. could be sourced directly from their database.

**Comment by BrendanTCC** [comment]
@wtspace I just had a look! am

**Comment by BrendanTCC** [comment]
Whoops... didnt mean to close this issue!
